fixbackend/dispatcher/dispatcher_service.py

```diff
@@ -85,7 +109,7 @@ async def process_cloud_account_changed_message(self, message: Json, context: Me
             case "cloud_account_created":
                 await self.cloud_account_created(CloudAccountId(message["cloud_account_id"]))
             case "cloud_account_deleted":
-                await self.cloud_account_deleted(CloudAccountId(message["cloud_account_id"]))
+                pass  # we don't care about deleted accounts since the scheduling is done via the tenant id
             case _:
                 log.error(f"Don't know how to handle messages of kind {context.kind}")
```

Comment:
Do we actually still need this?
- when a cloud account is connected, triggering the collect could be done by fixbackend.